GruntTasks/Options/concat.orchestrajs.js

```diff
@@ -91,6 +90,8 @@ module.exports = {
         'web/built/openorchestrabackoffice/js/table/TableviewCollectionView.js',
         'web/built/openorchestrabackoffice/js/table/tableviewLoader.js',
 
+        'web/built/openorchestrabackoffice/js/viewConfigurator.js',
```

Comment:
@juchi : Can you add a comment code in the same format as other sections please ?